app/views/moodSelection.html

```diff
@@ -1,13 +1,12 @@
 <div class="col-md-offset-3 col-md-6">
-    <div ui-view class="row marketing">
```

Comment:
This was the nested ui-view that had to be removed. As the `row` class is already existing on the outer `ui-view` there is no need to redefine it here, especially not *in* a column.